src/lib.rs

```diff
@@ -1110,6 +1110,65 @@ impl<K, V, S> OrderMap<K, V, S>
         });
     }
 
+    /// Sort the map’s key-value pairs by the default ordering of the keys.
+    pub fn sort_keys(&mut self)
```

Comment:
Perhaps a `sort_keys_by` helper would be nice as well?

Reply:
Additionally, would `sort_values` and `sort_values_by` be good for completion?

Reply:
personal preference: I don't want API explosion and I doubt that would be used very often. Even when it is used, it would not have significant savings.

Benefit: if keys and values are comparable it *might* be possible to do an accidental footgun like:

```
h.sort_by(|(k1, k2, _, _)| k1.cmp(k2))
```

Needless to say, this would be annoying (IMO still not enough of a benefit to justify it though).

Reply:
I don't want API explosion either. And we need to keep some headroom for the (inevitable?) `sort_unstably` family. Given the crate in question here, preferring stable sort seems obvious.

Reply:
Be a good sport about it, but You said “Would be nice”! We should not fall into the would be nice trap. Gets in the way of the "this is my actual usecase" stuff :-)